app/assets/javascripts/comfortable_mexican_sofa/admin/modules/CategoryExpand.js

```diff
@@ -0,0 +1,76 @@
+define(['jquery','DoughBaseComponent'], function ($, DoughBaseComponent) {
+'use strict';
+
+  var defaultConfig = {
+    categoriesCharacterCount: 20
+  };
+  var CategoryExpandProto;
```

Comment:
Small thing but it would be a good idea to combine these into one var statement...

```
var defaultConfig = {
       categoriesCharacterCount: 20
   },
   CategoryExpandProto;
```